docs/schema.md

```diff
@@ -11,7 +11,11 @@ Kubernetes namespace, service accounts, and so on. Your schema definition is
 used to create the form that users see when they deploy the application from
 the GCP Console.
 
-The format of `schema.yaml` follows a subset of JSON schema specifications.
+The format of `schema.yaml` follows a subset of JSON schema specifications. You
```

Comment:
Hmm, it's a bit tricky to organize this file. Let me take a stab at it, send you a copy and you can do further edits.

Reply:
Edits merged!